kafkakewl-common/src/main/scala/com/mwam/kafkakewl/common/http/EndpointUtils.scala

```diff
@@ -13,36 +13,48 @@ import sttp.tapir.EndpointIO.annotations.*
 import zio.json.{JsonDecoder, JsonEncoder}
 
 trait EndpointUtils {
-  val apiEndpoint: PublicEndpoint[Unit, Unit, Unit, Any] = endpoint.in("api" / "v1")
+  val apiEndpoint: PublicEndpoint[Unit, Unit, Unit, Any] =
+    endpoint.in("api" / "v1")
```

Comment:
By default `maxColumn = 80`, should we use something larger? I'd say at least 120, but I'm fine with more.

(I'm not a fan of too many line-breaks, less code fits in the screen)

Reply:
Done